geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/NullRedisSortedSet.java

```diff
@@ -49,4 +51,19 @@ long zadd(Region<RedisKey, RedisData> region, RedisKey key, List<byte[]> members
   byte[] zscore(byte[] member) {
     return null;
   }
+
+  @Override
+  byte[] zincrby(Region<RedisKey, RedisData> region, RedisKey key, byte[] increment,
+      byte[] member) {
+    byte[] incr = Coder.doubleToBytes(processIncrement(Coder.bytesToString(increment)));
```

Comment:
the double that's returned from `processIncrement()` is used in `RedisSortedSet`. Returning a string here would require two implementations of the process method. and I'd be more inclined to make the method void, versus just passing through the string unchanged

Reply:
The String might be changed by the method though, since there's some logic around the +/- infinity values that can take e.g. an input of `"+inf"` and turn it into a double value of `POSITIVE_INFINITY`, which then gets converted by `Coder.doubleToString()` into `"Infinity"`. Two implementations might be worth it, possibly "processIncrementToDouble" for the use in `RedisSortedSet` and "normalizeIncrement" or something for the use here, where we don't actually care about getting a double, just getting bytes that are consistent with the way we handle infinities and trailing `".0"` elsewhere.

Reply:
ok, I'll add that second implementation

Reply:
It seems confusing, at first glance, that we have 2 very similar implementations. I'd prefer to have a single one but we can adjust later if necessary. Bear in mind that the usage in `NullRedisSortedSet` is only going to be called once per key (on creation) but the other one will be called for all subsequent `zincrby`s on that key.